source/beamline/BioXAS/BioXASSSRLMonochromator.h

```diff
@@ -62,9 +62,11 @@ class BioXASSSRLMonochromator : public BioXASMonochromator
 	/// Returns the brake status control.
 	AMControl* brakeStatusControl() const { return brakeStatus_; }
 	/// Returns the bragg control.
-	AMControl* braggControl() const { return bragg_; }
+	AMControl* braggControl() const { return stepBragg_; }
```

Comment:
There may still be interest in the encoder-based bragg information, so I was advised to keep it available. The braggControl() getter is meant to return the preferred option (should be step-based bragg), but it may still be useful to ask explicitly for either option.

Historically, the encoder-based bragg motor control was the default (and named bragg_). It should have been renamed to encoderBragg_ once the stepBragg_ control was introduced, but wasn't everywhere. I've attempted to make clearer which was being used by renaming these old instances. Does this answer your question?